lib/fog/aws/models/storage/directory.rb

```diff
@@ -22,6 +22,13 @@ def acl=(new_acl)
           end
         end
 
+        def clear!
```

Comment:
Do we intend for this to only really be used inside of `destroy!`? I think that probably makes the most sense, and if so maybe we should move it down into the private methods to make that clearer. What do you think?

Reply:
Actually, I was thinking that it could be used standalone, that's why I left it public, but yeah, we can move it to private if you think it's better.

Reply:
I'm open to it being public, I just thought it might be confusing/surprising since it won't do multiple pages/retries and because people might then not be sure how to differentiate between when to use this and destroy. So yeah, seemed like maybe private was simpler/safer, but I'm certainly open to discussing it. Thanks!